addons/binding/org.openhab.binding.netatmo/ESH-INF/thing/welcomeCamera.xml

```diff
@@ -0,0 +1,113 @@
+            <channel id="welcomeCameraVpnUrl" typeId="vpn_url"></channel>
+            <channel id="welcomeCameraIsLocal" typeId="is_locale"></channel>
+                                    
+        </channels>
+              
+        <properties>
+             <property name="actualApp">app_welcomecamera</property>
+        </properties>         
+        
+        <config-description>
+            <parameter name="welcomeHomeId" type="text" required="true" readOnly="true">
+            	<label>Home ID</label>
+            	<description>Id of the home</description>
+            </parameter>
+
+            <parameter name="welcomeCameraId" type="text" required="true" readOnly="true">
+            	<label>Person ID</label>
+            	<description>Id of the camera</description>
+            </parameter>
+
+        </config-description>      
+	                          
+    </thing-type>
+
+                  
+    <channel-type id="id">
+        <item-type>String</item-type>
+        <label>Id</label>
+        <description>ID of the camera</description>
+        <state readOnly="true">
+        </state>
+    </channel-type>
+    <channel-type id="type">
+        <item-type>String</item-type>
+        <label>Type</label>
+        <description>Type of the camera</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    <channel-type id="status">
+        <item-type>String</item-type>
+        <label>State</label>
+        <description>State of the camera</description>
+        <state readOnly="true">
+            <options>
+                <option value="on">on</option>
+                <option value="off">off</option>
+            </options>
+        </state>       
+    </channel-type>
+    <channel-type id="sd_status">
+        <item-type>String</item-type>
+        <label>SD State</label>
+        <description>State of the SD card</description>
+        <state readOnly="true">
+            <options>
+                <option value="on">on</option>
+                <option value="off">off</option>
+            </options>
+        </state>   
+    </channel-type>
+    <channel-type id="alim_status">
+        <item-type>String</item-type>
+        <label>Alim State</label>
+        <description>State of the power connector</description>
+        <state readOnly="true">
+            <options>
+                <option value="on">on</option>
+                <option value="off">off</option>
+            </options>
+        </state>   
+    </channel-type>
+    <channel-type id="name">
+        <item-type>String</item-type>
+        <label>Camera name</label>
+        <description>Name of the camera</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    <channel-type id="vpn_url">
+        <item-type>String</item-type>
+        <label>VPN Url</label>
+        <description>Only for scope access_camera. Address of the camera</description>
```

Comment:
@watou Implemented the picture urls, but i haven't tested it with your PR #2412. But if you put the url in the url field of picture Item it worked. Would be nice if there would be an improvement to set the picture size also for the Picture ITEM.

I also implemented all Urls for the videos, but it seems that the Video item on sitemap couldn't stream m3u8 Videos, but the Links worked in VLC Media Client and in Safari browser, so they should be ok.